src/manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    "gcm_sender_id": "189059495005"
```

Comment:
what is the purpose of this exactly?

Is there any chance that we can set this programmatically with our init functions rather than having to add one piece of config separately in this file?

Reply:
As per my understanding, the `manifest.json` file configures the app to use Firebase Cloud Messaging (FCM) for push notifications, ensuring the web application to receive and manage notifications from Firebase's backend for the specific project. Additionally, this file needs to be included in the `index.html` script tag, allowing the Firebase backend to automatically detect the sender ID and send/receive notifications.

Let me try an alternative approach to set this up programmatically.

Reply:
I rechecked this by removing the `manifest.json` file and confirmed that we still receive the notification. However, we need to test this on the server to verify whether the notification is received or not. I need to check this in the PR deployment, but the deployed PR link has not been generated yet.